apps/web/src/views/Swap/SmartSwap/components/TransactionConfirmSwapContent.tsx

```diff
@@ -0,0 +1,126 @@
+import { Currency, CurrencyAmount, TradeType } from '@pancakeswap/sdk'
+import { TradeWithStableSwap } from '@pancakeswap/smart-router/evm'
+import { ConfirmationModalContent } from 'components/TransactionConfirmationModal'
+import { ITrade } from 'config/constants/types'
+import { memo, useCallback, useMemo } from 'react'
+import { Field } from 'state/swap/actions'
+import { maxAmountSpend } from 'utils/maxAmountSpend'
+import SwapModalFooter from '../../components/SwapModalFooter'
+import SwapModalHeader from '../../components/SwapModalHeader'
+import StableSwapModalHeader from '../../StableSwap/components/StableSwapModalHeader'
+import {
+  computeSlippageAdjustedAmounts as computeSlippageAdjustedAmountsWithSmartRouter,
+  computeTradePriceBreakdown as computeTradePriceBreakdownWithSmartRouter,
+} from '../utils/exchange'
+
+/**
+ * Returns true if the trade requires a confirmation of details before we can submit it
+ * @param tradeA trade A
+ * @param tradeB trade B
+ */
+function tradeMeaningfullyDiffers(tradeA: ITrade, tradeB: ITrade): boolean {
+  return (
+    tradeA.tradeType !== tradeB.tradeType ||
+    !tradeA.inputAmount.currency.equals(tradeB.inputAmount.currency) ||
+    !tradeA.inputAmount.equalTo(tradeB.inputAmount) ||
+    !tradeA.outputAmount.currency.equals(tradeB.outputAmount.currency) ||
+    !tradeA.outputAmount.equalTo(tradeB.outputAmount)
+  )
+}
+
+interface TransactionConfirmSwapContentProps {
+  trade: TradeWithStableSwap<Currency, Currency, TradeType> | undefined
+  originalTrade: ITrade | undefined
```

Comment:
I think if we have the `ConfirmSwapModal` component duplicated, we don't need `ITrade` type here anymore? This component simply works for trade with stable swap